politeiad/api/v1/v1.go

```diff
@@ -119,6 +119,10 @@ var (
 	ErrInvalidBase64 = errors.New("corrupt base64")
 	ErrInvalidMerkle = errors.New("merkle roots do not match")
 	ErrCorrupt       = errors.New("signature verification failed")
+
+	// Length of prefix of token used for lookups. This is a var so that it
+	// can be updated during testing.
+	TokenPrefixLength = 6
```

Comment:
github uses 7 digits. Why did we pick 6? do we need some sort of collision detection?

Reply:
Previous discussion about token length:
https://github.com/decred/politeia/pull/1048#discussion_r356139070

There is collision detection.

Reply:
@martonp I talked with marco about this.  Lets make it 7 so that it matches github's abbreviated hash length.  Please also update the comment to detail the reason behind the chosen length.